### PR TITLE
 :sparkles: New release - 0.21.1

### PR DESCRIPTION
**Pull request by ezrasilvera**
## Related issue(s)

Fixes #

### CODE REVIEW

File: docs/content/direct/release-notes.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ The following sections list the known issues for each release. The issue list is
 
 ## 0.21.0 and its release candidates
 
+### Patch release  0.21.1 
+This patch release mainly updates the documentation exposed under kubestellar.io . In all other aspects it is identical to 0.21.0
```
Comment (MikeSpreitzer) [inline docs/content/direct/release-notes.md:8]: > In all other aspects it is identical to 0.21.0

No, this is not true. There have been a few bug fixes.

I do not think that it is necessary to list them all here; GitHub automatically collects a changelog (see https://github.com/kubestellar/kubestellar/releases/tag/v0.21.0 for an example).  I might simply remove the inaccurate sentence.

File: docs/content/direct/release-notes.md [hunk 2]
```diff
@@ -4,6 +4,9 @@ The following sections list the known issues for each release. The issue list is
 
 ## 0.21.0 and its release candidates
 
+### Patch release  0.21.1
```
Comment (MikeSpreitzer) [inline docs/content/direct/release-notes.md:7]: "Patch release  0.21.1 " does not logically fit under "0.21.0 and its release candidates". BTW, even for a patch release we will have release candidates. I think the best approach is to simply have "## 0.21.1 and its release candidates".

**Comment by MikeSpreitzer** [comment]
This is not all that we need to do in order to prepare for a release.  See https://github.com/kubestellar/kubestellar/blob/main/docs/content/direct/release.md

File: docs/content/direct/release-notes.md (hunk 2 above)
Comment (ezrasilvera) [reply docs/content/direct/release-notes.md:7]: Done

**Review by MikeSpreitzer — COMMENTED** [review]
/lgtm
Thanks!

**Comment by MikeSpreitzer** [comment]
In double-checking, I did find this: https://github.com/kubestellar/kubestellar/blob/3df25e469cb8b6e45334ba7e3f40d488a79a9800/README.md?plain=1#L13

**Pull request merged**